File: OpenJudgeCN.Bailian/Quiz1000.cs
using System;

namespace OpenJudgeCN.Bailian
{
    public class Quiz1000
    {
        /*
        描述
        Calculate a + b
        
        输入
        Two integer a,,b (0 ≤ a,b ≤ 10)

        输出
        Output a + b

        样例输入
        1 2

        样例输出
        3
        */
        public static void Test()
        {
            var input = Console.ReadLine();
            var strParams = input.Split(" ");
            var a = Convert.ToByte(strParams[0]);
            var b = Convert.ToByte(strParams[1]);
            var res = a + b;
            Console.WriteLine(res);
        }
    }
}

File: Go/bailian/Quiz1000.go
package bailian

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

type Quiz1000 struct {
}

func (quiz *Quiz1000) GetIntroduce() string {
	return `
	描述
	Calculate a + b

	输入
	Two integer a,,b (0 ≤ a,b ≤ 10)
	输出
	Output a + b
	样例输入
	1 2
	样例输出
	3`
}

func (quiz *Quiz1000) Test() {
	var input string
	scanner := bufio.NewScanner(os.Stdin)
	if scanner.Scan() {
		input = scanner.Text()
	}
	inputs := strings.Split(input, " ")
	a, _ := strconv.Atoi(inputs[0])
	b, _ := strconv.Atoi(inputs[1])
	fmt.Println(a + b)
	return
}
